pyseir/load_data.py

```diff
@@ -553,183 +558,69 @@ def get_current_hospitalized_for_state(state: str, t0: datetime, category: Hospi
-
-    We initialize the model by taking the first X data points, calculating the average, and then
-    calculating steady state if historical data had matched current data. X is the average
-    length of stay. We then use that as the starting point to step forward in the model. So once we
-    have more than X datapoints, the forecast will shift based on underlying changes in the data.
-    Until we collect significant data, the estimates are sensitive to the inital values reported.
-    E.g. If day 1 shows 10 new ICU patients, we assume that has been happening for the last week
-    too and estimate accordingly. We can choose to wait for multiple points before extrapolating
-    to protect against surfacing noisy initial data to the user.
-
     Parameters
     ----------
-    cumulative: array
-        Array like sequence of daily cumulative values (expects, but doesn't enforce monotonic)
-    category: str
-        Either 'hospitalization' or 'icu
+    df
+        dataframe containing either current_ or cumulative_ values for a single county or state
+    t0
+        beginning of observation period
+    category
+        the type of current data to be returned
 
     Returns
     -------
-    current_estimate: float
-        Latest estimate of currently occupied beds.
+    time: float
+        Days since t0 for the hospitalization data.
+    current estimate: float
+        The most recent provided value for the current occupied in the requested category.
     """
-    average_length_of_stay = get_average_dwell_time(category)
-
-    # Calculate new admissions as the differences of daily cumulatives
-    daily_admits = np.diff(cumulative)
 
-    # When reporting starts, we assume there will already be a non-zero number of patients
-    # in the hospital/ICU. We need to estimate that starting value to initialize the model.
-    # If we don't, it takes ~2 times the average length of stay for the value to reach the
-    # expected. So our reported numbers for the first ~14 days would be too low (which we saw in
-    # Utah ICU data).
+    # TODO: No need to pass t0 down and back up. Can return a datetime that consumer converts.
 
-    # We average over the inputs for up to the same number of days as the average dwell
-    max_window = int(np.floor(average_length_of_stay))
-    initial_daily_estimate = daily_admits[:max_window].mean()
+    NUM_DAYS_TO_LOOK_BACK = 3
+    # Agencies will start and stop reporting values. Also, depending on the time of day some columns
+    # in a day row may propagate before others. Therefore, we don't want to just take the most
+    # recent value which may be None, nor take just the most recent any value, which may be weeks
+    # ago.
 
-    # And use that steady state solution as if that data had been entered in the past to initialize.
-    t0_patients = initial_daily_estimate * average_length_of_stay
+    # Datetimes are in UTC. Look at possible values that are within this time window.
```

Comment:
Oh, I thought they are naive (without TZ) so you'd want to compare them to a naive time to avoid doing a bunch of timezone aware comparisons. How about avoiding naive vs not by using dates? `datetime.date.today() - datetime.timedelta(days=3)`?

Reply:
Agree on simplifying dates. The combined dataset has UTC naive datetimes with the times set to 0s. So I need to roughly get the UTC date right now (which is often a "day" ahead of pacific time). That's why I call utcnow and then extract the date.